R/data.table.R

```diff
@@ -1497,7 +1497,7 @@ replace_dot_alias = function(e) {
   if (byjoin) {
     # The groupings come instead from each row of the i data.table.
     # Much faster for a few known groups vs a 'by' for all followed by a subset
-    if (!is.data.table(i)) stopf("logical error. i is not data.table, but mult='all' and 'by'=.EACHI")
+    if (!is.data.table(i)) stopf("logical error. i is not a data.table, but mult='all' and 'by'=.EACHI")
```

Comment:
this change is for making the two error messages consistent with each other "i is not a data.table"

Reply:
I did a grep and found that "not a data.table" is the more frequent wording (this error message was the only instance of "not data.table" without a)